README.md

```diff
@@ -1,101 +1,147 @@
-<h1 align="center">
-    fzf.fish
-</h1>
+<div align="center">
 
-A plugin that integrates [fzf](https://github.com/junegunn/fzf) into your [fish](http://fishshell.com) workflow. Comes with handy functions&mdash;each with its own mnemonic key binding&mdash;that use fzf to
```

Comment:
ah the reason I used &mdash is because it's a different character than just -. - is a bit too short and I think people are more used to seeing the longer dash. What do you think?

Reply:
If you take a look at my README versus yours (the rendered versions), the emdashes look the same. In many text editors however they appear almost exactly the same size.

Reply:
Oh I didn't realize that! Thanks